rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/RMRestInterface.java

```diff
@@ -765,4 +766,10 @@ String getStatHistory(@HeaderParam("sessionid") String sessionId, @QueryParam("r
     String getNodeThreadDump(@HeaderParam("sessionid") String sessionId, @QueryParam("nodeurl") String nodeUrl)
             throws NotConnectedException;
 
+    @GET
+    @GZIP
+    @Path("nodes/history")
+    @Produces("application/json")
+    List<RMNodeHistory> getNodesHistory(@HeaderParam("sessionid") String sessionId, @HeaderParam("windowStart") long windowStart,
```

Comment:
Very good job !
I suggest to return a Map<String, List<RMNodeHistory>> to ease the work from the client side.

Reply:
good. i will do it